services/dataplatform/client.go

```diff
@@ -0,0 +1,34 @@
+package dataplatform
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/meta"
+	dataplatform "github.com/ionos-cloud/sdk-go-dataplatform"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
+	"net/http"
+	"os"
+	"runtime"
+	"time"
+)
+
+// Client - wrapper over sdk client, to allow for service layer
+type Client struct {
+	sdkClient dataplatform.APIClient
+}
+
+func NewClient(username, password, token, url, version, terraformVersion string) *Client {
+	newConfigDataplatform := dataplatform.NewConfiguration(username, password, token, url)
+
+	if os.Getenv("IONOS_DEBUG") != "" {
+		newConfigDataplatform.Debug = true
+	}
+	newConfigDataplatform.MaxRetries = 999
```

Comment:
@cristiGuranIonos I noticed that these values are the same for every client, we should store them in a constant somewhere and reuse that, let's do this in another PR.

Reply:
we hvae the constants